entities/card_proprties.py

```diff
@@ -0,0 +1,42 @@
+from dataclasses import dataclass
+from datetime import datetime
+from typing import Dict
+from uuid import UUID
+
+from flask import Request
+
+from entities.card_type import CardType
+from consts import *
+
+
+@dataclass
+class CardProperties:
+    content: str
+    version: str
+    type: CardType
+    date: datetime
+    id: UUID
+
+    @staticmethod
+    def generate_card_properties(request: Request) -> 'CardProperties':
+        card_json = request.json
+        return CardProperties.generate_card_from_json(card_json)
+
+    @staticmethod
+    def generate_card_from_json(card_json: Dict[str, str]) -> 'CardProperties':
+        content = card_json[CONTENT]
```

Comment:
Consider "cleaning" this function a little bit... It's a bit long and defines variables that only used once

Reply:
function a bit long, but i dont think i can clean it that much
and i like the constistency of the inputs in CardProperties